src/upf/include/upf/upf.h

```diff
@@ -104,4 +104,15 @@ bool eval_upf(sta::dbNetwork* network,
               utl::Logger* logger,
               odb::dbBlock* block);
 
+bool update_power_switch_cell(utl::Logger* logger,
+                              odb::dbBlock* block,
+                              const char* name,
+                              const char* cell);
```

Comment:
dbMaster

Reply:
This one is read by the tcl interface so it has to stay char*

Reply:
done